frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -83,35 +85,47 @@ export default class Entity extends React.Component<Props> {
         return 'missing';
     }
 
-    selectEntity = (e: SyntheticMouseEvent<HTMLLIElement>) => {
-        // Flow requires that we use `e.currentTarget` instead of `e.target`.
-        // However in this case, we do want to use that, so I'm ignoring all
-        // errors Flow throws there.
-        // $FLOW_IGNORE
-        if (e.target && e.target.classList.contains('status')) {
-            return null;
-        }
-
+    selectEntity = () => {
         this.props.selectEntity(this.props.entity);
     };
 
+    getSiblingEntities = () => {
```

Comment:
We'll need to `stopPropagation` first. See Point 1 in https://github.com/mozilla/pontoon/pull/1866#discussion_r579644071.

Reply:
I have it set up like this passing the event object and then used `stopPropagation() `is this correct?

```    
getSiblingEntities = (e: SyntheticMouseEvent<HTMLSpanElement>) => {
        e.stopPropagation();
        this.props.getSiblingEntities(this.props.entity.pk);
    };
```

Reply:
Yeah, except it's not a span element the event is triggered on, it's a button. So you'll need to change the type (to `HTMLButtonElement` I think).